.github/workflows/ci-generate.yaml

```
@@ -0,0 +1,51 @@
+name: CI
+
+on:
+  pull_request:
+     branches:
+       - main
```

```suggestion
on:
  push:
    branches: [main]
  pull_request:
    branches: [main]
```
Run this on every commit to `main` in addition to every pull request.

---

Oh, why so? Won't this result in unnecessary workflow runs?

---

Honestly I'm not sure. I usually see this in `ci.yaml` workflows. Someone else may know better why this is worth doing, or if it's not.

---

This suggestion can be considered non-blocking for now since it's not critical.